client/valor/schemas/symbolic/annotations.py

```diff
@@ -0,0 +1,457 @@
+import io
+import typing
+import warnings
+from base64 import b64decode, b64encode
+from typing import Any, List, Optional, Union
+
+import numpy as np
+import PIL.Image
+
+from valor.enums import TaskType
+from valor.schemas.symbolic.atomics import (
+    Float,
+    MultiPolygon,
+    Nullable,
+    Polygon,
+    Spatial,
+    String,
+    Symbol,
+)
+
+
+class Score(Float, Nullable):
+    """
+    Implements the score annotation as a nullable floating-point variable.
```

Comment:
should we use our consistent docstring format for the docstrings in this file? seems like this one may be missing "Parameters"

Reply:
StaticCollection objects have parameters, Variables do not.